hawk/src/main/java/com/orhanobut/hawk/Hawk.java

```diff
@@ -137,6 +148,11 @@ public void call(Subscriber<? super Boolean> subscriber) {
     return null;
   }
 
+  private static boolean isHawkInitialised() {
```

Comment:
this should be public to the clients, they can also benefit from it. As method name, no need to put "hawk" word, since it's already inside Hawk class. It could be simply "isInitialised"